templates/rancher/overlay/opt/corral/rancher/preflight.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 set -ex
 
-apt install -y jq
-
 curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
 
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

Comment:
Could we potentially move this to it's own template?

Reply:
We could, but I think it's a nice to have kubectl installed when debugging or just view certain things in the cluster, but also it's not necessary as corral has functionality to get the kubeconfig file. It's your call, we can remove it altogether as well.